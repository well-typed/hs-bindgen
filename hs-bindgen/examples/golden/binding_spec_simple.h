typedef char sym;

struct pt {
  double x;
  double y;
};
