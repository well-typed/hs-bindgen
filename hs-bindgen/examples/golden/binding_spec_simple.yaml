version:
  hs_bindgen: 0.1.0
  binding_specification: '1.0'
types:
- omit:
    headers: binding_spec_simple.h
    cname: struct pt
- headers: binding_spec_simple.h
  cname: sym
  module: Generated
  hsname: Sym
  instances:
  - Bits
  - Bounded
  - Enum
  - Eq
  - FiniteBits
  - Integral
  - Ix
  - Num
  - Ord
  - Read
  - Real
  - Show
  - Storable
